### PR TITLE
Version 2.0.2

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @runderwood, just a simple patch bump here.

**Comment by ldko** [comment]
We could start a changelog file for this project.

**Comment by somexpert** [comment]
ah, yes, I'll add one

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Change Log
+==========
+
+2.0.2
+-----
+
+* Fixed bug that prevented password resets when no SITE_ID was set.
+* Corrected usage of deprecated setting.
+* Fixed sorting difference that occured after resending invitation email.
```
Comment (ldko) [inline CHANGELOG.md]: "occured" should be "occurred".

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+2.0.1
+-----
+
+* Fixed bug that prevented signing up for an accoung when no SITE_ID was set.
```
Comment (ldko) [inline CHANGELOG.md]: "accoung" -> "account"

Comment (somexpert) [inline CHANGELOG.md]: It's Monday morning, my internal spell-check hasn't woken up yet.

**Comment by runderwood** [comment]
ok. :+1:

**Pull request merged**